controllers/cloud.redhat.com/namespacereservation_controller.go

```diff
@@ -221,6 +223,8 @@ func (r *NamespaceReservationReconciler) Reconcile(ctx context.Context, req ctrl
 func (r *NamespaceReservationReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	return ctrl.NewControllerManagedBy(mgr).
 		For(&crd.NamespaceReservation{}).
+		Watches(&source.Kind{Type: &core.Namespace{}},
```

Comment:
Does the namespace reservation need to be triggered on changes to a namespace it owns? It seems like without extra logic in this controller it doesn't look like this addition performs anything new.